modules/post/windows/gather/credentials/mcafee_vse_hashdump.rb

```diff
@@ -20,8 +20,11 @@ def initialize(info = {})
       info,
       'Name'          => 'McAfee Virus Scan Enterprise Password Hashes Dump',
       'Description'   => %q(
-        This module extracts the password hash from McAfee Virus Scan
-        Enterprise (VSE) used to lock down the user interface.
+        This module extracts the password hash from McAfee Virus Scan Enterprise (VSE)
+        used to lock down the user interface. Hashcat supports cracking this type of 
+        hash using hash type sha1($salt.unicode($pass)) (-m 140) and a hex salt 
+        (--hex-salt) of 01000f000d003300 (unicode "\x01\x0f\x0d\x33"). A dynamic 
+        format is available for John the Ripper at the referenced URL.
```

Comment:
Travis build is failing because you have spaces at the end of some of the new lines.